python_modules/dagster/dagster/core/definitions/metadata/__init__.py

```diff
@@ -349,6 +351,29 @@ def emit_metadata(context, df):
 
         return IntMetadataValue(value)
 
+    @staticmethod
+    def boolean(value: bool) -> "BoolMetadataValue":
```

Comment:
I think it would be more consistent to call this `bool` than `boolean`, because we use `int` instead of `integer`.

Reply:
got it 👍 converted to `bool`